content/blog/entries/resource-gathering/contents.lr

```diff
@@ -0,0 +1,42 @@
+title: Resource Gathering
+---
+categories:
+outreachy
+design
+open-source
+---
+author: dhruvi16
+---
+series: ux-design-cycle
+---
+pub_date: 2020-06-05
+---
+body:
+
+
+<div style="text-align: center;">
+  <figure>
+    <img src="cover.jpeg"/>
+    <figcaption>
+      <em>
+        Photo by <a href="https://unsplash.com/@dandimmock?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Dan Dimmock</a>
+      </em>
+    </figcaption>
+  </figure>
+</div>
+
+I will be describing the UX design cycle with a series of articles and this article is about describing the first step of the design cycle which is Resource Gathering.
+
+The core concept of user experience design is that users use interfaces to accomplish a task. User experience design starts with the user, by user I mean the individual that is using some technology to accomplish some goal.
+
+The interface consists of an input and output through which the user interacts with the system. For instance, clicking a photo requires the user to press the button(input) and an image is the desired output. Creating an affordable and usable interface is the main goal of this process. Design is a data-driven process and resource gathering is all about gathering this data.
+
+The resource gathering process is about figuring out how the task is accomplished by the user currently. There are 4 ways to gather data and below I will describe them all in detail. There are two types of data — Quantitative(Numeric) and Qualitative(Thematic) and designers prefer to use both types of data as per requirement.
+
+1. Naturalistic observation - The goal of naturalistic observation is to watch the user in action in their environment. Rather than asking the user what, when, and where she accomplishes a given task, the designer goes out to where the task takes place and observes the user. Most naturalistic observations take place in the field rather than the lab because we want to get a sense of the environment where the task takes place. During the naturalistic observation session, the data collected can be either qualitative or quantitative. Qualitative data may be a set of notes written by the designer as he observes the user. Quantitative data may be collected during a naturalistic observation session as well.
```

Comment:
You could make the headings bolder to stand out.
```suggestion
1. **Naturalistic observation:** The goal of naturalistic observation is to watch the user in action in their environment. Rather than asking the user what, when, and where she accomplishes a given task, the designer goes out to where the task takes place and observes the user. Most naturalistic observations take place in the field rather than the lab because we want to get a sense of the environment where the task takes place. During the naturalistic observation session, the data collected can be either qualitative or quantitative. Qualitative data may be a set of notes written by the designer as he observes the user. Quantitative data may be collected during a naturalistic observation session as well.
```